tests/integration-service/integration.go

```diff
@@ -48,7 +48,7 @@ var _ = framework.IntegrationServiceSuiteDescribe("Integration Service E2E tests
 
 			applicationName = createApp(*f, testNamespace)
 			componentName, originalComponent = createComponent(*f, testNamespace, applicationName)
-			integrationTestScenario, err = f.AsKubeAdmin.IntegrationController.CreateIntegrationTestScenario(applicationName, testNamespace, BundleURL, InPipelineName)
+			integrationTestScenario, err = f.AsKubeAdmin.IntegrationController.CreateIntegrationTestScenario(applicationName, testNamespace, gitURL, revision, pathInRepoPass)
```

Comment:
I think we can finally remove the 2 commented lines below this one, can you please remove them as well?

Reply:
Also, we can remove the occurrence of `"github.com/redhat-appstudio/integration-service/api/v1alpha1"` from:
https://github.com/redhat-appstudio/e2e-tests/blob/a933a8585fc534aca8fcbd23025b33b9b9248f54/pkg/clients/kubernetes/client.go#L23
and
https://github.com/redhat-appstudio/e2e-tests/blob/a933a8585fc534aca8fcbd23025b33b9b9248f54/pkg/clients/kubernetes/client.go#L82
as well.

Rest LGTM

Reply:
Updated.

Reply:
Thank you!